### ISSUE TITLE
SAM returns masks with offset on large images

### ISSUE THREAD

**Issue by williamcorsel** [opencv/cvat#6374]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

The output polygons of SAM are placed with an y-axis offset when the image size is large (4400 x 2800px). This issue only occurs for object labelled in the lower parts of the image:

Object labelled in upper part of the image (mask is correct)
![cvat_sam_offset_upper](https://github.com/opencv/cvat/assets/31770711/592653ab-cdf0-4dcc-ba8c-1a0ad33c1ea6)

Object labelled in lower part of the image (mask has the correct shape but shifted upwards)
![cvat_sam_offset_lower](https://github.com/opencv/cvat/assets/31770711/db51331b-9b83-411d-9d24-ffeba27c8dcc)

This issue is consistant for all objects in the lower part of the image

### Steps to Reproduce (for bugs)

1. Load a large image into CVAT
2. Use SAM to segment objects

### Your Environment

- Git hash commit (`git log -1`): ec149df0d36a7278f6f7147baa223d6685483404
- Docker version `docker version` (e.g. Docker 17.0.05): 24.0.2
- Operating System and version (e.g. Linux, Windows, MacOS): Ubuntu
- Also affects https://app.cvat.ai/

**Comment by bsekachev** [comment]
Need to investigate

**Comment by bsekachev** [comment]
@williamcorsel 

I tried on the image 3589 x 5384 on app.cvat.ai and was not able to reproduce.
Could you please share a task on app.cvat.ai where it can be reproduced?

**Comment by williamcorsel** [comment]
@bsekachev Thanks for taking the time to look into this! I uploaded a backup of the task here: https://we.tl/t-M2r6lJ44lq.

**Comment by bsekachev** [comment]
@williamcorsel 

Thank you. As a temporary workaround you can just unpin and drag masks where you need:
<img width="168" alt="image" src="https://github.com/opencv/cvat/assets/40690378/6dd6d6a7-0747-4d41-a4b4-4faa001d4cf1">

**Comment by williamcorsel** [comment]
Yes that's very useful. I noticed that the keyboard shortcut for this function is missing. Is this by design?

**Comment by vignesh-Ag** [comment]
@bsekachev  this problem occurs on self-hosted version with lower resolution too, as mentioned in #6604.
